cabal.project

```diff
@@ -1,3 +1,15 @@
 -- HI! for pants reasons, if you add anything load bearing in here, you have to
--- add it to raw-project in cabal.haskell-ci
+-- add it to `generate cabal.project` in `.github/workflows/haskell-ci.yml`
+source-repository-package
+  type: git
+  location: https://github.com/nikita-volkov/refined
+  tag: 11296288b5e2f2b391ee721b50af3cfe7beaa790
+
+source-repository-package
+  type: git
+  location: https://github.com/TOTBWF/refinery
+  tag: e9dbb5fd990ff040c2496c7cf527c45550eb189b
```

Comment:
It makes me a little nervous to depend on upstream git patches for something we may want to push to Hackage - why are we depending on these here?

Reply:
the maintainers are incommunicado.

Reply:
but, yeah, i agree, this is an awful hack, and not really appropriate for a published library

Reply:
I think it may just be a spurious dependency? on `main`, if I remove the dep, all tests pass just fine